homeassistant/components/media_player/__init__.py

```diff
@@ -176,6 +178,24 @@ def is_on(hass, entity_id=None):
     {"type": WS_TYPE_MEDIA_PLAYER_THUMBNAIL, "entity_id": cv.entity_id}
 )
 
+WS_TYPE_MEDIA_PLAYER_BROWSE_MEDIA = "media_player/browse_media"
+SCHEMA_WEBSOCKET_BROWSE_MEDIA = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend(
```

Comment:
This is not how we define web socket command anymore. Check https://github.com/home-assistant/core/blob/dev/homeassistant/components/camera/__init__.py#L625-L632

Reply:
I had originally used that syntax but mypy was complaining. I think using both the old and new methods at the same time was causing problems. I'll try to also convert `media_player_thumbnail` to the new method and see if it stops complaining.